src/command.ts

```diff
@@ -53,7 +143,8 @@ function takeScreenshot(
     objToOperateOn
       .screenshot(name, {
         ...screenshotOptions,
-        onAfterScreenshot(_el: JQuery, props: OnAfterScreenshotProps) {
+        // eslint-disable-next-line
+        onAfterScreenshot(_el: JQuery, props: any) {
```

Comment:
I purposely let `any` here as I don't want to define an external type in our code, seems wrong

Reply:
When dealing with libraries it is common for library authors to type some props as `any` 👍